### PR TITLE
feat: init compendium

### PR DESCRIPTION
**Pull request by salmad3**
### Context:

This PR proposes the 'AtomOne Compendium', a **temporary document** designed to capture the early stage initiatives of AtomOne. It serves as a guide during the project's formative phase, offering an overview of ongoing efforts, discussions, and key themes.

- Helps avoid overcrowding in issue threads.
- Serves external community engagements.

The compendium includes:

- Summaries of key issues and their discussions.
- Categorization of issues under common themes such as of governance, tokenomics, and infrastructure.

A few improvements and edits will be made in a fast-follow update.
Please feel free to suggest better categorization and point out any details that should be included.

### CODE REVIEW

**Comment by Pipello** [comment]
We will soon need something like the https://forum.cosmos.network/ for those discussions

File: COMPENDIUM.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+[<ins>Issue #22</ins>](https://github.com/atomone-hub/genesis/issues/22) seeks to clarify the 
+relationship between AtomOne Hub's GitHub repository and the Tendermint repository.
+
+#### Key points:
+
+- Discussion on the relationship between AtomOne Hub's and Tendermint's repositories.
+- Need for a constitution and clarity on PHOTON design.
+
+### Should Atomone be an ICS Consumer for any Blockchain?
```
Comment (Pipello) [inline COMPENDIUM.md:220]:
```suggestion
### Should AtomOne be an ICS Consumer for any Blockchain?
```

File: COMPENDIUM.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+| [<ins>#22: Link with Tendermint Atom One</ins>](https://github.com/atomone-hub/genesis/issues/22)                    | moul                 | Questions about the connection between the AtomOne repository and Tendermint Atom One. Suggestions for a constitution and clarity on PHOTON design. |
+| [<ins>#23: Issues/PRs Moderation</ins>](https://github.com/atomone-hub/genesis/issues/23)                            | moul                 | Challenge of selecting a responsible party for moderation. Proposal of an operations committee for moderation decisions.                            |
+| [<ins>#24: Setup Weekly Report</ins>](https://github.com/atomone-hub/genesis/issues/24)                              | moul                 | Idea of initiating a weekly report. Use tools like ChatGPT to compile the list, with human intervention for fine-tuning.                      |
+| [<ins>#25: Should AiB Hire Dedicated Resources for Atomone?</ins>](https://github.com/atomone-hub/genesis/issues/25) | moul                 | Proposal to create a job for overseeing Atomone at AiB. Its role is to facilitate progress and act as a bridge between the community and AiB.                  |
```
Comment (Pipello) [inline COMPENDIUM.md:31]:
```suggestion
| [<ins>#25: Should AiB Hire Dedicated Resources for Atomone?</ins>](https://github.com/atomone-hub/genesis/issues/25) | moul                 | Proposal to create a job for overseeing AtomOne at AiB. Its role is to facilitate progress and act as a bridge between the community and AiB.                  |
```

File: COMPENDIUM.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+- Rewarding contributors and creators within the AtomOne ecosystem.
+
+### Official Cosmos ATOM-ONE NFT
+
+[<ins>Issue #20</ins>](https://github.com/atomone-hub/genesis/issues/20) explores creating an official 
+AtomOne NFT.
+
+#### Key points:
+
+- NFT as recognition for early Atom-One project supporters.
```
Comment (Pipello) [inline COMPENDIUM.md:312]:
```suggestion
- NFT as recognition for early AtomOne project supporters.
```

File: COMPENDIUM.md [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+
+#### Key points:
+
+- Debate on who should lead in moderating the GitHub repository.
+- Proposed structures for moderation and the use of tools like ChatGPT.
+
+### Setup Weekly Report
+
+[<ins>Issue #24</ins>](https://github.com/atomone-hub/genesis/issues/24) proposes creating a weekly report 
+for the Atom-One project.
```
Comment (Pipello) [inline COMPENDIUM.md:378]:
```suggestion
for the AtomOne project.
```

File: COMPENDIUM.md [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+
+- Proposal for AtomOne’s brand identity and Twitter handle reservation.
+- Move towards natural element motifs, focusing on the Hydrogen molecule.
+- Significance of Hydrogen in symbolizing fundamental properties.
+- Design proposals with community feedback and development suggestions.
+
+### ATOM-ONE Secure Communications for Users
+
+[<ins>Issue #21</ins>](https://github.com/atomone-hub/genesis/issues/21) highlights the need for secure 
+communication alternatives for ATOM-ONE users.
```
Comment (Pipello) [inline COMPENDIUM.md:272]:
```suggestion
communication alternatives for AtomOne users.
```

**Review by Pipello — COMMENTED** [review]
Besides issues title I think we should stick with one typo for **AtomOne**

**Review by giunatale — APPROVED** [review]
this is amazing! 
I think we need something like this. It's a great starting point and we'll eventually better understand also how to synthesize and collate points, as issues gets addressed and concepts ossified.

**Review by moul — COMMENTED** [review]
Awesome, 👏

**Pull request merged**